js/gamestate.js

```diff
@@ -19,7 +19,14 @@ const CARDS = [
     name: "Donk",
     description: "End the next two turns.",
     endTurn: 2,
-    cost: 0
+    cost: 7
+  },
+  {
+    name: "Moloch's Howl",
```

Comment:
I wasn't sure about the keyword "howl" or introducing proper nouns like "Moloc" just yet.

I'd say let's go for a space theme for now, with extremely practical names/keywords. 

Example Names:

 * TwoBot
 * Laser for a targeted damage spell
 * Errant Blast for a randomized terror.
 * Maybe Battleship for a creature that creates other creatures (that reminds me, I should make a card like this)
(FWIW I will to rename donk better too)

Example Keywords:
 * damage
 * kill
 * defend